examples/specs/interactive_concat.vl.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "Show a line chart corresponding to selected points.",
+  "data": {
+    "values": [
+      {"id": "001", "ra": "243.35", "dec": "+54.6", "time": "2", "mag": 19},
+      {"id": "001", "ra": "243.35", "dec": "+54.6", "time": "1", "mag": 18.5},
+      {"id": "002", "ra": "210.35", "dec": "+14.6", "time": "2", "mag": 20},
+      {"id": "002", "ra": "210.35", "dec": "+14.6", "time": "1", "mag": 19.5},
+      {"id": "003", "ra": "143.35", "dec": "+33.6", "time": "2", "mag": 18},
+      {"id": "003", "ra": "143.35", "dec": "+33.6", "time": "1", "mag": 19}
+    ]
+  },
+  "vconcat": [
+    {
+      "width": 300,
+      "height": 200,
+      "title": "Sky position",
+      "transform": [{"aggregate": [], "groupby": ["ra", "dec", "id"]}],
```

Comment:
I wonder if the concept you wanna show is more just this aggregate with groupby only.  (Maybe worth adding a simpler example to the aggregate page instead? Or at least add another one.)

Reply:
I don't think aggregate without group by is not that magical.

Reply:
I don't think aggregate with only group by is that magical and not the main point of this example.